### DISCUSSION TITLE
Visual Studio Marketplace

### DISCUSSION THREAD

**Discussion by ulissescappato** [picoe/Eto#2576] [General]
Hi!

Will the new version be updated on the Visual Studio Marketplace?

Thanks,
Ulisses.

**Comment by cwensley** [comment]
They are fully up to date, other than the included Eto, but it doesn't stop you from updating your projects to use the latest.  One is for VS 2019 (x86), and one is for VS 2022 (x64 and arm64).  See https://marketplace.visualstudio.com/search?term=eto.forms&target=VS&category=All%20categories&vsVersion=&sortBy=Relevance

**Reply by ulissescappato** [reply]
Thanks, Curtis!
